src/pages/organizations/pages/organizations-page/index.tsx

```diff
@@ -147,17 +151,23 @@ const OrganizationsPage: FC<Props> = ({
         selectedCategory={currentCategory}
         onCategoryChange={onCategoryChange}
       />
-      <div className='row mb-5'>
+      <SC.CheckBoxAndLinkWrapper>
         <CheckBox
           id='showAllCheckbox'
           active={!includeEmptyOrganizations}
           onClick={toggleShowEmptyOrganizations}
           textLabel={
             localization.organizationsPage.onlyShowOrganizationsWithContent
           }
-          displayClass='col-12'
+          displayClass=''
```

Comment:
```suggestion
```

Reply:
Kan dette fikses senere? Er i merge to main"/ Kan lage issue på det

Reply:
Nei, vent var det at jeg fikk en feilmelding på at den manglet className, så satte den til tom streng. Mulig det fins en fiks, men vil gjerne ta det i en egen commit/PR, rekker ikke fikse nå.